gitapp/lib/user_details/user_info_widget.dart

```diff
@@ -0,0 +1,102 @@
+        Column(
+          children: <Widget> [
+            SizedBox(height: 20.0),
+            Container(
+              padding: EdgeInsets.all(4.0),
+              child: Text(
+                'INFO',
+                style: TextStyle(
+                  fontSize: 20.0,
+                  color: yyellow,
+                  letterSpacing: 1.5,
+                ),
+              ),
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(6.0),
+                border: Border.all(width: 1.0, color: yyellow),
+              ),
+            ),
+            SizedBox(height: 15.0),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                Column(
+                  children: [
+                    Text(
+                      'Followers',
+                      style: TextStyle(
+                        fontSize: 13.0,
+                        letterSpacing: 2.0,
+                      ),
+                    ),
+                    SizedBox(height: 5.0),
+                    Text(
+                      userMock.followersCount.toString(),
+                      style: TextStyle(
+                        fontSize: 11.0,
+                        letterSpacing: 1.0,
+                        color: yyellow,
+                      ),
+                    ),
+                  ],
+                ),
+                SizedBox(width: 10.0),
+                Column(
+                  children: [
+                    Text(
+                      'Repositories',
+                      style: TextStyle(
+                        fontSize: 13.0,
+                        letterSpacing: 2.0,
+                      ),
+                    ),
+                    SizedBox(height: 5.0),
+                    Text(
+                      userMock.repositoriesCount.toString(),
+                      style: TextStyle(
+                        fontSize: 11.0,
+                        letterSpacing: 1.0,
+                        color: yyellow,
+                      ),
+                    ),
+                  ],
+                ),
+                SizedBox(width: 10.0),
+                Column(
+                  children: [
+                    Text(
+                      'Country',
+                      style: TextStyle(
+                        fontSize: 13.0,
+                        letterSpacing: 2.0,
+                      ),
+                    ),
+                    SizedBox(height: 5.0),
+                    Text(
+                      userMock.country,
+                      style: TextStyle(
+                        fontSize: 11.0,
+                        letterSpacing: 1.0,
+                        color: yyellow,
+                      ),
+                    ),
+                  ],
+                ),
+              ],
+            ),
+          ],
+        ),
+      ]
+  );
```

Comment:
100 lines for one widget? 
Maybe we could extract it a little bit more?

Reply:
It has to stay that way because it's one widget and it doesn't make sense to separate it.